modules/nf-core/bcftools/pluginsplit/main.nf

```diff
@@ -42,7 +43,19 @@ process BCFTOOLS_PLUGINSPLIT {
         ${targets_arg} \\
         --output ${prefix}
 
-    mv ${prefix}/* .
+    if [ -n "${suffix}" ]; then
+        for file in ${prefix}/*; do
+            # Extract the basename
+            base_name=\$(basename "\$file")
+            # Extract the part of the basename before the first dot
+            name_before_dot="\${base_name%%.*}"
+            # Extract the extension
+            extension="\${base_name#\${name_before_dot}}"
+            # Construct the new name
+            new_name="\${name_before_dot}${suffix}\${extension}"
+            mv "\$file" "${prefix}/\$new_name"
```

Comment:
What happens here if there are dots in the sample names?

Reply:
Okay, so that's effectively a problem...
I will change the way the splitting is done to only check for (vcf, bcf, gz, tbi, csi) then append the suffix and the extension afterwards.

Reply:
See my other comment :)